### ISSUE TITLE
Not detecting missing aria-label on input field

### ISSUE THREAD

**Issue by alzapiedi** [component-driven/cypress-axe#14]
My test file: 
```
context('app', () => {
  beforeEach(() => {
    cy.server();
    cy.route('GET', '/api/exercises/core/exercises/*', 'fixture:exercises');
    cy.visit('/exercises/3bc94905-4b8a-4d67-815e-0652afaf22c2');
  });

  it('has no a11y violations', () => {
    cy.injectAxe();
    cy.checkA11y();
  });
});
```

the output (note the input field shown in element explorer)
<img width="1542" alt="Screen Shot 2019-08-27 at 12 11 08 PM" src="https://user-images.githubusercontent.com/11680925/63788737-fbc9e000-c8c3-11e9-8f9a-582d2274c636.png">

`A11Y ERROR! landmark-one-main on 1 Node`
`A11Y ERROR! page-has-heading-one on 1 Node`

shows nothing about missing label on input, am i doing something wrong?  ive tried adding arguments to the `checkA11y` call to make it more specific but then i get no errors at all
